docs/index.md

```diff
@@ -13,13 +13,65 @@ myst:
 
 This is the community-maintained documentation for the Plone content management system.
 
-This documentation is for:
+:::::{grid} 1 1 2 2
+:gutter: 3 3 4 5
 
-- Editors who write and update content on a site
-- Site administrators who install Plone and add-ons, and set up a site
-- Designers who create site themes
-- Deployers who configure servers for site hosting
-- Developers who customize a site’s capabilities, create add-ons, and contribute to Plone itself
+::::{grid-item-card} 
+:link: https://6.docs.plone.org/contributing/index.html
+{octicon}`book;1.5em;sd-text-info`
+Contributing guide
+^^^
+This part of the documentation describes how to contribute to Plone, including all its projects and repositories under the Plone GitHub organization.
++++
+```{button-link} https://6.docs.plone.org/contributing/index.html
+:color: primary
+:expand:
+Go to Contributing guide
+```
+::::
+
+::::{grid-item-card} 
+:link: https://6.docs.plone.org/install/index.html
+{octicon}`zap;1.5em;sd-text-info`
+Get started
+^^^
+This part of the documentation helps you find the best way to get started with Plone, depending on what you want to do.
++++
+```{button-link} https://6.docs.plone.org/install/index.html
+:color: primary
+:expand:
+Go to Get started
+```
+::::
+
```

Comment:
I think Get Started should be at the start. Let's move this to the first card.

I think we should also add a "User guide" card as the second card. We will need a new page as well, `user-guide/index.md`. It should include a description and two links.

1. https://6.docs.plone.org/volto/user-manual/index.html
2. [Content Editing for Plone](https://training.plone.org/content-editing/index.html).